taxonomies/ingredients.txt

```diff
@@ -27137,6 +27165,10 @@ sv:glukossirap från vetemjöl
 # openfoodfacts:https://world.openfoodfacts.org/ingredient/fr:sirop-de-glucose-de-blé-en-poudre
 # 49 products in 3 languages @2018-09-24
 
+<en:glucose syrup
+en:dried glucose syrup
```

Comment:
this entry is unneeded, taken care of by ingredients processing taxonomy

Reply:
But it doesn't pass 
[(https://es.openfoodfacts.org/cgi/test_ingredients_analysis.pl?ingredients_text=jarabe+de+glucosa+deshidritado&type=add&action=process&.submit=Submit)]